.moai/reports/t232/sync-audit-wrapscan.py

```
@@ -0,0 +1,51 @@
+        continue
+    line = lines[hit_idx]
+    pos = line.index(clause) + len(clause)
+    rest = line[pos:].strip()
+    if rest:
+        midline.append((e['id'], 'rest=%r' % rest[:40]))
+        continue
+    if clause.rstrip()[-1:] in TERM:
+        continue
+    nxt = next((line for line in lines[hit_idx + 1:] if line.strip()), '')
+    if nxt:
+        wrap.append((e['id'], 'clause-tail=%r' % clause[-25:], 'next=%r' % nxt.strip()[:45]))
```

Ruff (0.16.2)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)